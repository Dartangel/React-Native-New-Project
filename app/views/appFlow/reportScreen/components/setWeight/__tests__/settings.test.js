import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Weight from '../index';

describe('Weight', () => {
    test('Weight snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<Weight />);
        expect(snapshot).toMatchSnapshot();
    });
});