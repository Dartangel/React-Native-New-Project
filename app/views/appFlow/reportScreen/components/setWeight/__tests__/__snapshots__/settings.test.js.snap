// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weight Weight snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#ffffff",
        "height": 300,
        "width": 300,
      }
    }
  >
    <InputValue
      header="Weight"
    />
    <InputValue
      header="Height"
    />
    <View
      style={
        Object {
          "flex": 0.3,
          "flexDirection": "row",
          "justifyContent": "space-around",
        }
      }
    >
      <OwnBtn
        title="Cansel"
      />
      <OwnBtn
        title="Save"
      />
    </View>
  </View>
</View>
`;
